integration-tests/data/simple-circle/circle/snapcraft.yaml

```diff
@@ -0,0 +1,53 @@
+name: real-circle
+summary: A big bad circle
+description: These don't really make sense for tests
```

Comment:
Add a longer description here for the test (comment I once got from @elopio) like

```yaml
description: |
  The objective of this test is to blah ...
  more description ... blah blah